protocols/803-somagenics/bead_cleanup.ot2.py

```diff
@@ -0,0 +1,62 @@
+from opentrons import labware, instruments, modules
+
+# labware setup
+mag_deck = modules.load('magdeck', '2')
+plate = labware.load('96-flat', '1')
+pcr_strips = labware.load('96-PCR-flat', '2', share=True)
```

Comment:
Best to change this to biorad-hardshell-96-PCR plate. This is the one we are currently supporting for magnetic module